pkg/store/models/transaction.go

```diff
@@ -11,6 +12,16 @@ import (
 	api "github.com/trisacrypto/trisa/pkg/trisa/api/v1beta1"
 )
 
+const (
+	SourceLocal    = "local"
+	SourceRemote   = "remote"
+	StatusDraft    = "draft"
+	StatusPending  = "pending"
+	StatusAction   = "action required"
+	StatusComplete = "completed"
+	StatusArchived = "archived"
+)
```

Comment:
These are the list of source and status constant values available for transaction source and status fields.